jekyll/_cci2/workflows.md

```diff
@@ -0,0 +1,205 @@
+- Fan-in for deployment to separate platforms with automated triggers 
+- Branch-level job execution with artifact sharing
+
+For example, you might want to run acceptance tests independently from integration tests and deployment. Use workflows to orchestrate parts of your build and to increase your ability to respond to failures. Scheduled jobs appear in the Workflows tab of the CircleCI app, so you have an integrated view of the status of every individual workflow as shown in the following screenshot. 
+
+![CircleCI Workflows Page]({{ site.baseurl }}/assets/img/docs/workflow_landing.png)
+
+Select the Failed workflow to see the status of each job as shown in the next screenshot. Click the Rerun button and select From beginning to restart a failed workflow or select From failed to restart a failed job.
+
+![CircleCI Workflows Page]({{ site.baseurl }}/assets/img/docs/workflow_detail.png)
+
+## Parallel Job Execution Example
+
+The basic Workflows configuration runs all jobs in parallel. That is, jobs listed in the `workflows` section without additional keys will run at the same time. The blue icons indicate the Running status.
+
+![Parallel Job Execution Workflow]({{ site.baseurl }}/assets/img/docs/parallel_jobs.png) 
+
+To run a set of parallel jobs, add a new `workflows:` section to the end of your existing configuration file with the version and a unique name for the workflow. The following `.circleci/config.yml` file snippet shows an example workflow with four parallel jobs. It is defined by using the `workflows:` key named `build-and-test` and by nesting the `jobs:` key with a list of job names. The jobs have no dependencies defined, therefore they will run in parallel.
+
+```
+workflows:
+  version: 2
+  build-and-test:
+    jobs:
+      - build
+      - test1
+      - test2
+      - test3
+```
+
+## Sequential Job Execution Example
+
+The following example shows a workflow with four sequential jobs. The jobs run according to configured  requirements, each job waiting to start until the required job finishes successfully as illustrated in the diagram. 
+
+![Sequential Job Execution Workflow]({{ site.baseurl }}/assets/img/docs/sequntial_workflow.png)
+
+The following `config.yml` snippet is an example of a workflow configured for sequential job execution:
+
+```
+workflows:
+  version: 2
+  build-test-and-deploy:
+    jobs:
+      - build
+      - test1:
+          requires:
+            - build
+      - test2:
+          requires:
+            - test1
+      - deploy:
+          requires:
+            - test2	
+```
+
+The dependencies are defined by setting the `requires:` key as shown. The `deploy:` job will not run until the `build` and `test` jobs complete successfully. A job must wait until all upstream jobs in the dependency graph have run. So, the `deploy` job waits for the `test2` job, the `test2` job waits for the `test1` job and the `test1` job waits for the `build` job.
```

Comment:
The second sentence mentions a `test` job which isn't present in the config. Might be clearer to drop the `test2` job and rename `test1` to `test`.